src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -50,11 +61,21 @@ public void processHearingRequest(HearingWrapper wrapper) throws UnhandleableHea
                 break;
             default:
                 UnhandleableHearingState err = new UnhandleableHearingState(wrapper.getState());
-                log.error(err.getMessage(),err);
+                log.error(err.getMessage(), err);
                 throw err;
         }
     }
 
+    public HearingResponse sendDeleteHearingRequest(HearingWrapper wrapper) {
+        HearingDeleteRequestPayload payload = buildDeleteHearingPayload(wrapper);
+        return hmcHearingApi.deleteHearingRequest(
+            idamService.getIdamTokens().getIdamOauth2Token(),
+            idamService.getIdamTokens().getServiceAuthorization(),
+            idamService.getIdamTokens().getUserId(),
+            payload
```

Comment:
Nitpick.. don't need local variable can just call method here

Reply:
I thought it just cleaner tbh but will move it under the method.